voting/src/voting.rs

```diff
@@ -101,9 +113,53 @@ impl Context {
                 self.storage_mut().set(vote_paper.vote_paper_id.0.as_ref(), serde_cbor::to_vec(&vote_paper).unwrap());
                 Ok(vote_paper.vote_paper_id)
             }
+            VOTE_TX_TYPE => {
+                let tx: VoteTransaction =
+                    serde_cbor::from_slice(&transaction.body()).map_err(|_| ExecuteError::InvalidFormat)?;
+                let vote_paper_id = tx.action.vote_paper_id;
+                let choice = tx.action.choice;
+                let voter_signature = tx.action.voter_signature;
+
+                let mut vote_paper =
+                    self.get_vote_paper(&vote_paper_id).map_err(|_| ExecuteError::VotePaperNotFound)?;
+
+                if !vote_paper.verify_signature(&voter_signature) {
+                    return Err(ExecuteError::InvalidVoterSignature)
+                }
+
+                let vote = Vote::new(vote_paper_id, choice, voter_signature);
+                let vote_id = vote.vote_id.clone();
+                vote_paper.set_used_in(vote_id.clone()).map_err(|_| ExecuteError::UsedVotePaper)?;
+                self.storage_mut().set(vote_id.as_ref(), serde_cbor::to_vec(&vote).unwrap());
+
+                let meeting_id = vote_paper.get_general_meeting_id();
+
+                let mut vote_box = self.meeting_mut().get_vote_box(&meeting_id).unwrap();
+                vote_box.drop_in_box(vote.vote_id.clone());
+
+                self.meeting_mut().update_vote_box(&meeting_id, &vote_box);
```

Comment:
I missed this. Please do not ignore returned Result.

```suggestion
                self.meeting_mut().update_vote_box(&meeting_id, &vote_box).expect("Votebox is created when creating a general meeting.");
```